### ISSUE TITLE
Certificate creation and cert/key storage

### ISSUE THREAD

**Issue by cbrit** [PeggyJV/steward#40]
With the use of TLS we need a signed server certificate and client CA root cert with which to issue a signed client cert to the Data Science team. The keys and certs need to be stored securely. 

We're leaving the larger process of issuing new certs and onboarding Data Providers to future work.

**Comment by cbrit** [comment]
Currently waiting on the Data Science team to generate and provide client certs
